protocol/x/sending/keeper/msg_server_create_transfer.go

```diff
@@ -113,8 +116,22 @@ func (k msgServer) SendFromModuleToAccount(
 	ctx := sdk.UnwrapSDKContext(goCtx)
 
 	if err := k.Keeper.SendFromModuleToAccount(ctx, msg); err != nil {
+		telemetry.IncrCounterWithLabels(
```

Comment:
This may not be a big save for total lines of code, but FYI you can use named parameters with a defer pattern to have a single telemetry call in code, see [example](https://github.com/dydxprotocol/v4-chain/pull/603/files)

Reply:
Ah I see! Will keep that in mind for the future.